benchkit/commandattachments/klockstat.py

```diff
@@ -143,6 +144,15 @@ def attachment(
             current_dir=rdd,
         )
 
+        # Wait until the clock stat has at least outputted something in the out file, or the error file,
+        # in order to know that it has attached the eBPF.
+        while True:
```

Comment:
i'm afraid of infinite loop in case of errors. Should we try 100 times and then raise an error?